src/expandProperty.js

```diff
@@ -1,41 +1,57 @@
-const WHITESPACE_NO_CALC = /\s+(?=[^)]*?(?:\(|$))/g
-const LENGTH_UNIT = /(calc\(|em|ex|ch|rem|vw|vh|vmin|vmax|cm|mm|q|in|pt|pc|px|dpi|dpcm|dppx|%|auto)$/i
```

Comment:
The `calc(` check here was definitely not working since the regular expression ends with `$`.